### ISSUE TITLE
Mixed case tags not added to checklist in v2.0.2

### ISSUE THREAD

**Issue by CAI-Alice** [delashum/obsidian-checklist-plugin#115]
I recently updated to v2.0.2 and checklist stopped working for me - it would sit with the spinning/waiting symbol forever, and not add any of my items to the list. My tag that I used successfully with the previous version was #ToDo. I did some experiments and found v2.0.2 is able to list checklist items with a lowercase (#todo) tag. Not sure if this is related to a breaking change regarding custom styles mentioned for one of the previous releases - I'm using the 'Minimal' theme, but have made no other style changes.

Love that you're able to use multiple tag names in v2.0.2 so I'm looking forward to being able to update!

The plugin is awesome and a massive part of my daily workflow - thank you for all the hard work!

**Comment by rolfaalders** [comment]
Indeed. Had the same issue( ToDo instead of todo), but did an bulk update the tags to lower case and plugin started working again.

**Comment by drxmknight** [comment]
Please add Case sensitive option. Thanks!
